app/attributes/rememberattributescontroller.h

```diff
@@ -68,9 +72,10 @@ class  RememberAttributesController : public QObject
     };
 
     bool mRememberValuesAllowed = false;
-    //! Remembered last created feature for each layer (key)
-    QHash<QString, RememberedValues> mRememberedValues;
+    //! Remembered last created feature for each project/layer (key)
+    static QHash<QString, RememberedValues> sRememberedValues;
```

Comment:
why there is need for static variable?

Reply:
A `RememberAttributesController` is instantiated with each forms' `AttributeController`, so we need the storage to persist. We agreed with a static var instead of QSettings for now.